### ISSUE TITLE
3-3 Tasks - Add lecture notes to materials

### ISSUE THREAD

**Issue by soniamitchell** [IBAHCM/RPiR#16]
If students didn't make notes during the lecture.. then this:

"you need to sample from the binomial distribution (as described in the lecture) to determine the number of new infecteds and susceptibles "

might cause them all to ask for help.

It might be worth summarising lecture key points in the materials (I haven't looked at any of your slides yet)

**Comment by richardreeve** [comment]
Yes, I think we want to be as independent of the lecture materials as possible (and ideally really just get rid of the lecture materials completely and teach from the docs in the end).
